[Profiler][Trivial] Switch to nanoseconds for Result's internal representation

robieta opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #77697
* #77696
* #77695
* #77694

Certain steps in building the call tree rely on sorting, so we want to retain as much precision as possible. `profiler_kineto.cpp` and KinetoEvent still use microseconds.

Differential Revision: [D36302563](https://our.internmc.facebook.com/intern/diff/D36302563/)

Commit:
[Profiler][Trivial] Switch to nanoseconds for Result's internal representation

Certain steps in building the call tree rely on sorting, so we want to retain as much precision as possible. `profiler_kineto.cpp` and KinetoEvent still use microseconds.

Differential Revision: [D36302563](https://our.internmc.facebook.com/intern/diff/D36302563/)

[ghstack-poisoned]

aaronenyeshi reviewed (approved):
Reviewed internally, LGTM!

robieta commented:
@pytorchbot merge this please

Commit:
Update on "[Profiler][Trivial] Switch to nanoseconds for Result's internal representation"


Certain steps in building the call tree rely on sorting, so we want to retain as much precision as possible. `profiler_kineto.cpp` and KinetoEvent still use microseconds.

Differential Revision: [D36302563](https://our.internmc.facebook.com/intern/diff/D36302563/)

[ghstack-poisoned]

Commit:
Update on "[Profiler][Trivial] Switch to nanoseconds for Result's internal representation"


Certain steps in building the call tree rely on sorting, so we want to retain as much precision as possible. `profiler_kineto.cpp` and KinetoEvent still use microseconds.

Differential Revision: [D36302563](https://our.internmc.facebook.com/intern/diff/D36302563/)

[ghstack-poisoned]

robieta commented:
@pytorchbot merge this please
